assets/queries/k8s/service_account_token_automount_not_disabled/test/negative2.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: redistest-sa
+automountServiceAccountToken: false
+---
+metadata:
```

Comment:
We need to add a kind here so that the k8s validator passes

Reply:
Done, thank you for the hint!